packages/@webex/contact-center/src/cc.ts

```diff
@@ -0,0 +1,33 @@
+import {WebexPlugin} from '@webex/webex-core';
+import {CCConfig, IContactCenter, WebexSDK} from './types';
+import {WCC_API_GATEWAY} from './constants';
+
+export default class ContactCenter extends WebexPlugin implements IContactCenter {
+  namespace = 'WebexCC';
```

Comment:
Question.. what does this namespace do and where do we utilize it?

Reply:
Not sure, but it was being used in all the plugins. Will check and update here